### PR TITLE
Fixed unloading functionality for Blowgun

### PR DESCRIPTION
**Pull request by Marsunpaisti**
Blowpipe unload teleported items to weird positions, now unloads on unit loc properly if items dont fit into inventory

### CODE REVIEW

File: wurst/_items/Blowgun.wurst [hunk 1]
```diff
@@ -169,11 +158,8 @@ class Blowpipe
             updateTooltip()
 
     function loadAmmo(unit owner, item itm)
-        if not this.loadingEnabled
-            return
-
         int charges = itm.getCharges()
```
Comment (jonathanmclaus) [inline wurst/_items/Blowgun.wurst]: `int charges = max(itm.getCharges(), 1)`

Comment (jonathanmclaus) [reply wurst/_items/Blowgun.wurst]: See if `itm.getCharges() or 1` works. I'm guessing Wurst does not support typing like that, but worth a shot.

Comment (Marsunpaisti) [reply wurst/_items/Blowgun.wurst]: I think the former is clearer

File: wurst/_items/Blowgun.wurst [hunk 2]
```diff
@@ -158,10 +158,8 @@ class Blowpipe
             updateTooltip()
 
     function loadAmmo(unit owner, item itm)
-        int charges = itm.getCharges()
         //Treat non-charge items as 1 charge
-        if charges == 0
-            charges = 1
+        int charges = max(itm.getCharges(),1)
```
Comment (jonathanmclaus) [inline wurst/_items/Blowgun.wurst]: You're missing a space. Did you try using `or`?

Comment (Marsunpaisti) [reply wurst/_items/Blowgun.wurst]: yes

File: wurst/_items/Blowgun.wurst [hunk 3]
```diff
@@ -13,10 +12,11 @@ import InventoryEvent
...
 import ID
+import UnitExtensions
 
 constant let ICON = Icons.bTNAlleriaFlute
 constant let ITEM_TOOLTIP = "Purchase |cffffcc00B|rlow Gun"
-constant let ITEM_TOOLTIP_EXT = "Used for shooting small things. Can be loaded with thistles or bones. Drag and drop items on the blowpipe to load, click and use on your own troll to unload."
+constant let ITEM_TOOLTIP_EXT = "Used for shooting small things. Can be loaded with thistles or bones. Drag and drop items on the blowgun to load, click and use on your own troll to unload."
 constant let ITEM_DESCRIPTION = "A hollowed out piece of wood. If blown through, could be used to fire small projectiles such as thistles."
 constant let GROUND_MODEL_PATH = "Doodads\\Northrend\\Water\\FloatingPlank\\FloatingPlank.mdl"
 constant let ABILITY_ID = 'XBPA'
```
Comment (jonathanmclaus) [inline wurst/_items/Blowgun.wurst]: Reference this variable when you create the item.

Comment (Marsunpaisti) [reply wurst/_items/Blowgun.wurst]: done

**Pull request merged**